fadecandy_driver/src/fadecandy_driver.h

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) 2021 Eurotec
+//
+
+#include <array>
+#include <libusb-1.0/libusb.h>
+#include <vector>
+#include <iostream>
+
+constexpr int LEDS_PER_PACKET = 21;
+constexpr int LOOKUP_VALUES_PER_PACKET = 31;
+constexpr int LOOKUP_VALUES_PER_CHANNEL = 257;
+constexpr int USB_PACKET_SIZE = 64;
+constexpr int PACKET_TYPE_VIDEO = 0x00;
+constexpr int PACKET_TYPE_LUT = 0x40;
+constexpr int FINAL_PACKET_BIT = 0x20;
+
+constexpr int USB_PRODUCT_ID = 0x607a;
+constexpr int USB_VENDOR_ID = 0x1d50;
+constexpr int USB_ENDPOINT = 1;
+constexpr int INTERFACE_NO = 0x01;
+
+constexpr int LEDS_PER_STRIP = 64;
+constexpr int NUM_STRIPS = 8;
+
+namespace fadecandy_driver
+{
+//!
+//! \brief The FadecandyDriver class
+//!
+class FadecandyDriver
+{
+public:
+  //!
+  //! \brief The color struct contains the r,g and b colors
+  //!
+  struct Color
+  {
+    double r;
+    double g;
+    double b;
+  };
+  libusb_device* fadecandy_device_ = NULL;
```

Comment:
do these all need to be members?

Reply:
@JaccovdS  @reinzor These variables are device specific each variable of these is needed in different parts/functions in the program that's why I declared them as members variables.